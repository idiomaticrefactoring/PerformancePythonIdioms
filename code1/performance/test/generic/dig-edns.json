[{"id":24245,"opcode":"QUERY","status":"NOERROR","flags":["qr","rd","ra"],"query_num":1,"answer_num":1,"authority_num":0,"additional_num":1,"opt_pseudosection":{"edns":{"version":0,"flags":["do"],"udp":4096},"cookie":"b5dafd2648fa583b"},"question":{"name":"www.google.com.","class":"IN","type":"A"},"answer":[{"name":"www.google.com.","class":"IN","type":"A","ttl":177,"data":"216.58.195.132"}],"query_time":53,"server":"2600:1700:bab0:d40::1#53(2600:1700:bab0:d40::1)","when":"Thu Apr 15 10:19:50 PDT 2021","rcvd":59,"when_epoch":1618507190,"when_epoch_utc":null}]
