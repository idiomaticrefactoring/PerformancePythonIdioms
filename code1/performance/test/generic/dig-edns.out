; <<>> DiG 9.10.6 <<>> www.google.com
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 24245
;; flags: qr rd ra; QUERY: 1, ANSWER: 1, AUTHORITY: 0, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags: do; udp: 4096
; COOKIE: b5dafd2648fa583b
;; QUESTION SECTION:
;www.google.com.            IN  A

;; ANSWER SECTION:
www.google.com.     177 IN  A   216.58.195.132

;; Query time: 53 msec
;; SERVER: 2600:1700:bab0:d40::1#53(2600:1700:bab0:d40::1)
;; WHEN: Thu Apr 15 10:19:50 PDT 2021
;; MSG SIZE  rcvd: 59

